.github/workflows/publish.yml

```diff
@@ -3,6 +3,9 @@ on:
   push:
     branches:
       - main
+  pull_request:
```

Comment:
@pngwn Do you think it's ok to run this workflow on each PR as we discussed?

Reply:
No, we can only run the publish when we want to trigger the publish workflow, this will also fail for third part contributions as they won't have access to secrets. We need to expose the assets in one of the other workflows, the deploy space workflow is probably the most appropriate.

Reply:
> We need to expose the assets in one of the other workflows, the deploy space workflow is probably the most appropriate.

OK, so you mean
1. to move `pnpm pack` to the install-deps action
2. and to use it from deploy-spaces workflow then expose the artifacts
?
(Same question: is it ok to add such steps to the deploy-space workflow without renaming it?)

Reply:
Yep. And yeah it's fine to add it to the deploy space workflow without renaming.

Reply:
Thanks, so I applied the changes but `pnpm pack` is not added to the install-deps action as it's now only needed in one place.